src/Grid.js

```diff
@@ -0,0 +1,133 @@
+import Cell from './Cell.js';
+
+export default class Grid
+{
+	constructor()
+	{
+		this.cycle = 0;
+		this.cells = [];
+		this.cyclePositions = [];
+	}
+
+	/**
+	 * @param {number} xPosition
+	 * @param {number} yPosition
+	 * @return {string}
+	 */
+	getCellKey(xPosition, yPosition)
+	{
+		return `${xPosition},${yPosition}`;
+	}
+
+	/**
+	 * @param {number} xPosition
+	 * @param {number} yPosition
+	 * @param {Cell} cell
+	 * @param {Grid}
+	 */
+	setCell(xPosition, yPosition, cell)
+	{
+		let cellKey = this.getCellKey(xPosition, yPosition);
+		this.cells[cellKey] = cell;
+
+		if (!cell.isAlive()) {
+			return this;
+		}
+
+		for (let xIndex = xPosition - 1; xIndex <= (xPosition + 1); xIndex++) {
+			for (let yIndex = yPosition - 1; yIndex <= (yPosition + 1); yIndex++) {
+				cellKey = this.getCellKey(xIndex, yIndex);
+				if (typeof this.cells[cellKey] === 'undefined') {
+					this.cells[cellKey] = new Cell(false);
+				}
+			}
+		}
+
+		return this;
+	}
+
+	/**
+	 * @param {number} xPosition
+	 * @param {number} yPosition
+	 * @returns {Cell}
+	 */
+	getCell(xPosition, yPosition)
+	{
+		let cellKey = this.getCellKey(xPosition, yPosition);
+		if (typeof this.cells[cellKey] === 'undefined') {
+			return new Cell(false);
+		}
+
+		return this.cells[cellKey];
+	}
+
+	nextCycle()
+	{
+		let nextCycleCellsArguments = [];
+
+		for (var key in this.cells) {
```

Comment:
@dominiktilp I kinda fall in the impression it does not really matter, since array is stored as object as well. But for readability purposes, I agree.